tpgtools/product.go

```diff
@@ -29,8 +31,27 @@ func NewProductMetadata(packagePath, productName string) *ProductMetadata {
 	}
 }
 
+func (pm *ProductMetadata) WriteBasePath() bool {
+	return !pm.SkipBasePath()
```

Comment:
Why call & negate here rather than just putting the logic in this method? I'd prefer consolidating the logic unless SkipBasePath is called elsewhere